packages/material-ui/src/ButtonBase/TouchRipple.test.js

```diff
@@ -76,38 +74,45 @@ describe('<TouchRipple />', () => {
         },
         cb,
       );
-      expect(container.querySelector('.ripple').style).to.have.property('height', '1px');
-      expect(container.querySelector('.ripple').style).to.have.property('width', '1px');
+      expect(queryRipple().style).to.have.property('height', '1px');
+      expect(queryRipple().style).to.have.property('width', '1px');
     });
   });
 
   it('should create individual ripples', () => {
-    const { instance, container } = createTouchRipple();
+    const { instance, queryAllActiveRipples, queryAllStoppingRipples } = renderTouchRipple();
 
-    checkRipples(container, 0, 0);
+    expect(queryAllActiveRipples()).to.have.lengthOf(0);
+    expect(queryAllStoppingRipples()).to.have.lengthOf(0);
 
     instance.start({ clientX: 0, clientY: 0 }, cb);
-    checkRipples(container, 1, 0);
+    expect(queryAllActiveRipples()).to.have.lengthOf(1);
+    expect(queryAllStoppingRipples()).to.have.lengthOf(0);
 
     instance.start({ clientX: 0, clientY: 0 }, cb);
-    checkRipples(container, 2, 0);
+    expect(queryAllActiveRipples()).to.have.lengthOf(2);
+    expect(queryAllStoppingRipples()).to.have.lengthOf(0);
 
     instance.start({ clientX: 0, clientY: 0 }, cb);
-    checkRipples(container, 3, 0);
```

Comment:
Named parameters would've already helped a lot. LMK if the current approach is too verbose for readability then we can use named instead.

Positional is too error prone. At least if you have structurally equal arguments next to each other (see modal aria-hidden bug).

Reply:
I thought about using named parameters but I never made the jump since my IDE reveals argument names next to all of the parameters. Probably would have made sense to do so.